### PR TITLE
feat: rewrite

### PR DESCRIPTION
**Pull request by Marsup**

### CODE REVIEW

**Comment by Nargonath** [comment]
There is a problem on the API page when Carbon is displayed. It's over the API section and we can't read anything:

image

I'm on a small 13" laptop so that could be the reason (1280x573)

**Review by Nargonath — COMMENTED** [review]
I like what you did overall. I skimmed through the files quickly but you put order in files, changed formatting, move from sass to postcss. I like it.

**Comment by Nargonath** [comment]
I'm not a fan of the borders everywhere to be honest, but that's my personal taste:

image

**Comment by Nargonath** [comment]
nav items seem to be bigger and have more spacing. Here is the current joi.dev:

image

Here is the result on this branch:

image

We should probably go back to previous values IMO.

**Comment by Nargonath** [comment]
Twitter link seems to still point to sidwaytech: 
image

**Comment by Nargonath** [comment]
We lost the padding on the modules cards as well, they look better on current joi.dev.

Same if you compare https://joi.dev/module/pinpoint/api/?v=2.0.1 with the equivalent on this branch.

**Comment by Nargonath** [comment]
When I run `npm run generate` it skips a lot of versions for the changelog, is that normal?

image

I didn't dig into the code, I'm just asking.

File: .dockerignore [hunk 1]
```diff
@@ -1,5 +0,0 @@
-Dockerfile
```
[the remark this answers is not in the document]
Comment (Marsup) [reply .dockerignore:1]: Yes, it's not that useful IMHO, we're not going to use it in production, and it causes friction. I don't want an unnecessary maintenance burden.

**Comment by Marsup** [comment]
> Twitter link seems to still point to sidwaytech

Should we point to hapi's account? Considering joi itself has no account, I'm not sure what to do here.



> We lost the padding on the modules cards as well, they look better on current joi.dev.
> 
> Same if you compare https://joi.dev/module/pinpoint/api/?v=2.0.1 with the equivalent on this branch.

I'm not sure what you mean, could you show what's wrong?



> When I run `npm run generate` it skips a lot of versions for the changelog, is that normal?

Yup, I did my best not to regenerate versions that were already previously generated before to speed up the generation process. From now on, if a version is wrong, you'll have to remove it (or everything) from the json files.

**Comment by Nargonath** [comment]
Sorry @Marsup I didn't have time to review it unfortunately.

**Pull request merged**